src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -1,23 +1,29 @@
 (function(global, doc, eZ, bootstrap) {
-    const notificationsContainer = doc.querySelector('.ez-notifications-container');
+    const notificationsContainer = doc.querySelector('.ibexa-notifications-container');
     const notifications = JSON.parse(notificationsContainer.dataset.notifications);
     const template = notificationsContainer.dataset.template;
+    const iconsMap = {
+        info: 'system-information',
+        error: 'circle-close',
+        warning: 'warning-triangle',
+        success: 'checkmark',
+    };
     const addNotification = ({ detail }) => {
-        const { onShow, label, message, rawPlaceholdersMap = {} } = detail;
-        const templateLabel = label === 'error' ? 'danger' : label;
+        const { onShow, label, message, customIconPath, rawPlaceholdersMap = {} } = detail;
         const config = eZ.adminUiConfig.notifications[label];
         const timeout = config ? config.timeout : 0;
         const container = doc.createElement('div');
+        const iconPath = customIconPath ?? eZ.helpers.icon.getIconPath(iconsMap[label]);
```

Comment:
What do you think about change:
```suggestion
        const iconPath = customIconPath || eZ.helpers.icon.getIconPath(iconsMap[label]);
```
for `customIconPath` equal to `null`?

Reply:
I think this is the case with `??` or do you mean something else?
ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator